eth/deployer/rollup/rollup.go

```diff
@@ -35,7 +37,7 @@ const (
 	DefaultMinGasLeft             = 10_000
 	DefaultMaxTxsPerCommit        = 32
 
-	originalCostEstimatorAddress = "079d8077c465bd0bf0fc502ad2b846757e415661"
+	originalCostEstimatorAddress = "9f19c6456854fda109dfc2b9ad45d9bf147c8557"
```

Comment:
hubble-contracts does a keyless deploy ("Nick's method") to ensure that the cost estimator is always deployed to exactly the same address on every chain and the rest of the contracts hardcode that address.

hubble simply deploys the cost estimator. In order to get the rest of the contracts to work messes with their bytecode before deploying them: it searches their bytecode for `originalCostEstimatorAddress` and replaces it with the actual address.

The hubble-contracts update changed the hardcoded address so this needs to be changed to match.